core/src/main/java/org/pentaho/platform/scheduler2/quartz/QuartzScheduler.java

```diff
@@ -237,16 +237,21 @@ public static Trigger createQuartzTrigger( IJobTrigger jobTrigger, QuartzJobKey
           "QuartzScheduler.ERROR_0001_FAILED_TO_SCHEDULE_JOB", jobId.getJobName() ), e ); //$NON-NLS-1$
       }
     } else if ( jobTrigger instanceof SimpleJobTrigger ) {
-      SimpleJobTrigger simpleTrigger = (SimpleJobTrigger) jobTrigger;
-      long interval = simpleTrigger.getRepeatInterval();
-      if ( interval > 0 ) {
-        interval *= 1000;
+      try {
```

Comment:
Adding exception handling here similar to above block for complex triggers; ensures helpful errors from quartz get shown to the user.